chatwoot/src/ChatwootPlugin.php

```
@@ -0,0 +1,47 @@
+use Filament\Forms\Components\TextInput;
+use Filament\Notifications\Notification;
+use Filament\Panel;
+
+class ChatwootPlugin implements HasPluginSettings, Plugin {
+    use EnvironmentWriterTrait;
+
+    public function getId(): string {
+        return 'chatwoot';
+    }
+
+    public function register(Panel $panel): void {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$panel'. (undefined)

(UnusedFormalParameter)